### DISCUSSION TITLE
Possibility to preload track data before song is played

### DISCUSSION THREAD

**Discussion by oliver-hekmatara** [gilbarbara/react-spotify-web-playback#174] [Ideas]
From what I can see, there's no way to preload track data without actually playing a song. It would be great to have the possibility to load data like the image, artist, and title before the user has clicked on play.

**Comment by gilbarbara** [comment]
Hey @oliver-hekmatara 

Added in 0.14.5 🎉

**Reply by oliver-hekmatara** [reply]
That's great, thanks! 🙏
